www/front_src/src/Resources/Details/tabs/Timeline/index.tsx

```diff
@@ -166,7 +166,7 @@ const TimelineTab = ({ details }: TabProps): JSX.Element => {
     reload();
   }, [selectedTypes]);
 
-  const changeSelectedTypes = (_, typeIds): void => {
+  const changeSelectedTypes = (e, typeIds): void => {
```

Comment:
Shouldn't this be caught by ESLint as unused variable?

Reply:
I forgot to align the related frontend-core's PR to master

Reply:
I've updated the frontend-core's PR and it seems that Eslint does not catch this as unused variable

Reply:
Here is the explanation : https://eslint.org/docs/rules/no-unused-vars#args